### ISSUE TITLE
Exception in GetTCDataCommand.ts when tcModel is null

### ISSUE THREAD

**Issue by nealrosen04** [InteractiveAdvertisingBureau/iabtcf-es#81]
**Version**
1.0.0-beta.8

**Module**
cmpapi

**Describe the bug**
The getTCData command does not seem to properly handle the case where cmpApi.tcModel is set to null.

A javascript exception is thrown in the following code when trying to access tcModel.gvl because tcModel is null.
```
private sendData(): void {
    const callback = this.callback as TCDataCallback;
    const tcModel = CmpApiModel.tcModel as TCModel;

    if (!tcModel.gvl && !CmpApiModel.tcString) {
    ...
```

**Steps to Reproduce**
1. Set cmpApi.tcModel to null to signify that gdpr does not apply
2. Call the getTCData command 

**Expected behavior**
The command's callback function should be called with a TCData object that reflects the TCF spec response for when GDPR does not apply.

**Actual behavior**
A script exception occurs preventing the callback from being called.

**Comment by chrispaterson** [comment]
Definitely see it...

**Comment by nealrosen04** [comment]
Hi Chris.  I see you have a branch with a potential fix for this.  If this change is ready, could you please merge it?

**Comment by chrispaterson** [comment]
@nealrosen04 I released this fix in 1.0.0-alpha.14
